src/components/cta-button-link.tsx

```diff
@@ -0,0 +1,26 @@
+import { Link } from 'react-router-dom'
+import styled from 'styled-components/macro'
+
+export const CtaButtonLink = styled(Link)`
+  align-items: center;
+  transition: background 200ms linear;
+  font-size: ${props => props.theme.sizes.buttonText};
+  font-family: ${props => props.theme.fontFamily};
+  font-weight: 500;
+  border-radius: 28px;
+  padding: calc(${props => props.theme.sizes.buttonText} * 0.75) 24px;
+  box-shadow: 0 5px 20px rgba(0, 0, 0, 0.1);
+  min-width: 6em;
+  margin: 3px;
+  color: #fff;
+  text-decoration: none;
+  background: #079af3;
+  display: block;
+  text-align: center;
+  flex-shrink: 0;
+  &:hover {
+    opacity: 0.7;
+  }
+`
```

Comment:
Also to note when we make them more 'generalized' --> we can export all of the buttons as `styled.button`, then used SC's polymorphic [withComponent](https://styled-components.com/docs/api#withcomponent) factory if you want to use it as a link

```ts
export const CtaButton = styled.button`...`

// somewhere where you need it as a Link:
import { CtaButton } from './buttons'

CtaLink = CtaButton.withComponent(Link)
```

Reply:
Very cool. We should definitely take some time for this. I didn't want to go too deep on the cleanup, but this one was bugging me